src/checkers/inference/solver/backend/logiql/LogiQLSolver.java

```diff
@@ -80,7 +80,12 @@ public LogiQLSolver(SolverEnvironment solverEnvironment, Collection<Slot> slots,
     }
 
     @Override
-    public void convertAll() {
+    public Collection<Constraint> explainUnsatisfiable() {
+        return null;// Doesn't support right now
```

Comment:
As discussion in this PR, should use empty collection represent not support this feature.